.drone.yml

```diff
@@ -40,9 +40,9 @@ steps:
     BOT_USERNAME: kite-bot
     BOT_NAME: Kite Bot
     BOT_EMAIL: kite-bot@heliostech.fr
-    BRANCH_NAME: ${DRONE_BRANCH}
+    BRANCH_NAME: master
     REPO_NAME: ${DRONE_REPO}
```

Comment:
Please, close this PR and create integrate step for telegram and slack notification  in this PR https://github.com/openware/barong/pull/974